capabilities/connect.capabilities.yml

```diff
@@ -139,6 +139,170 @@ properties:
                 type: string
                 examples:
                 - Ethernet1
+  mclag-capabilities:
+    description: Capabilities of the MC-LAG (Multi-Chassis Link Aggregation) feature in the switch
+    type: object
+    properties:
+      max-mclag-groups:
+        description: Defines the maximum number of MC-LAG groups that can be configured on the switch.
+        type: integer
+      max-ports-per-mclag-group:
+        description: Specifies the maximum number of physical ports that can be part of a single MC-LAG group.
+        type: integer
+      max-vlans-per-mclag-group:
+        description: Indicates the maximum number of VLANs that can be supported within a single MC-LAG group.
+        type: integer
+      peer-link-bandwidth:
+        description: Defines the required minimum bandwidth for the peer link that connects two MC-LAG switches.
+        type: string
+        enum:
+          - 1G
+          - 10G
+          - 40G
+          - 100G
+      dual-active-detection:
+        description: Describes the dual-active detection mechanism to prevent both switches from becoming active simultaneously.
+        type: string
+        enum:
+          - ICCP
+          - Backup-Link
+          - None
+      failover-time:
+        description: Specifies the time required for traffic to fail over to the secondary switch when there is a failure in the primary switch.
+        type: integer
+        unit: "milliseconds"
+      vlan-synchronization:
+        description: Indicates whether VLAN synchronization across MC-LAG peers is supported and the number of VLANs that can be synchronized.
+        type: boolean
+      max-mac-entries-per-mclag:
+        description: Maximum number of MAC address entries that can be synchronized across MC-LAG peers.
+        type: integer
+  lldp-capabilities:
+    type: object
+    description:
+      Description of LLDP capabilities across different switch models/vendors.
+    properties:
+      supported-tlvs:
+        type: array
+        items:
+          type: string
+          enum:
+          - management-ip-address
+          - port-description
+          - system-capabilities
+          - system-description
+          - system-name
+          - proto-ident
+          - proto-vid
+          - pvid
+          - vlan-name
+          - link-agg
+          - mac-phy
+          - max-frame
+          - poe
+          - med-location
+          - med-inventory
+          - med-capabilities
+          - med-network-policy
+      lldp-med-support:
+        type: boolean
+        description: Specifies whether LLDP-MED support is available on the switch.
+        examples:
+        - true
+        - false
+      max-neighbors:
+        type: integer
+        description: Maximum number of LLDP neighbors a switch can discover and maintain.
+        examples:
+        - 64
+        - 128
+        - 256
+      refresh-interval:
```

Comment:
maybe rename to refresh-interval-seconds?
As a general comment going forward, not just for this change, if there is a time related value, specify the unit (seconds, milliseconds, minutes, etc) in the name would immediately add clarity